irods/manager/metadata_manager.py

```diff
@@ -21,18 +21,24 @@ class InvalidAtomicAVURequest(Exception): pass
 
 class MetadataManager(Manager):
 
+    @property
+    def use_timestamps(self):
+        return getattr(self,'_use_ts',False)
+
     __kw = {}  # default (empty) keywords
 
     def _updated_keywords(self,opts):
         kw_ = self.__kw.copy()
         kw_.update(opts)
         return kw_
 
-    def __call__(self, admin = False, **irods_kw_opt):
-        x = copy.copy(self)
-        irods_kw_opt.update([() if not admin else (kw.ADMIN_KW,"")])
-        x.__kw = irods_kw_opt
-        return x
+    def __call__(self, admin = False, timestamps = False, **irods_kw_opt):
+        if admin:					# logic changed to fix
+            irods_kw_opt.update([(kw.ADMIN_KW,"")])	#   issue #389
```

Comment:
agreed, a real 'why' or a test or nothing at all.

Reply:
Yes, I'll just take the comment out.  #389 is implicitly tested in  `test_timestamp_access_386` (in `meta_test.py`) anyway, because the line 
```
meta_ts = meta(timestamps = True)
```
omits the admin keyword.

Reply:
perfect.